src/controllers/v1/review-stages.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * name : review-stages.js
+ * author : Priyanka Pradeep
+ * created-date : 29-July-2024
+ * Description : Controller for review stage details.
+ */
+
+const reviewStagesService = require('@services/review-stages')
+const common = require('@constants/common')
+const utils = require('@generics/utils')
+
+module.exports = class reviewStages {
+	/**
+	 * update review stage
+	 * @method
+	 * @name update
+	 * @param {Object} req - request data.
+	 * @returns {JSON} - review stages data
+	 */
+
+	async update(req) {
+		try {
+			let organization_id = req.decodedToken.organization_id
```

Comment:
@priyanka-TL Can you confirm below logic - System admin can update/list for any org, org admin can do so only for their orgs.

Reply:
@aks30 for system admin org id will take from token hence he can only update his org details. For admin he pass org id in query, or he can update his org details